server/routes/user.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2022 Google LLC
```

Comment:
Should we add a /admin subdirectory under routes for all admin / user related pages?

Reply:
We don't have subdirectory for pages (only has /api to hold all apis), looks like each file handles routes for one page. Maybe keep this as is to be consistent?